src/cli/version.zig

```diff
@@ -34,32 +32,37 @@ pub fn run(alloc: Allocator) !u8 {
     try stdout.print("  - channel: {s}\n", .{@tagName(build_config.release_channel)});
 
     try stdout.print("Build Config\n", .{});
-    try stdout.print("  - Zig version: {s}\n", .{builtin.zig_version_string});
-    try stdout.print("  - build mode : {}\n", .{builtin.mode});
-    try stdout.print("  - app runtime: {}\n", .{build_config.app_runtime});
-    try stdout.print("  - font engine: {}\n", .{build_config.font_backend});
-    try stdout.print("  - renderer   : {}\n", .{renderer.Renderer});
-    try stdout.print("  - libxev     : {s}\n", .{@tagName(xev.backend)});
+    try stdout.print("  - Zig version   : {s}\n", .{builtin.zig_version_string});
+    try stdout.print("  - build mode    : {}\n", .{builtin.mode});
+    try stdout.print("  - app runtime   : {}\n", .{build_config.app_runtime});
+    try stdout.print("  - font engine   : {}\n", .{build_config.font_backend});
+    try stdout.print("  - renderer      : {}\n", .{renderer.Renderer});
+    try stdout.print("  - libxev        : {s}\n", .{@tagName(xev.backend)});
     if (comptime build_config.app_runtime == .gtk) {
-        try stdout.print("  - desktop env: {s}\n", .{@tagName(internal_os.desktopEnvironment())});
-        try stdout.print("  - GTK version:\n", .{});
-        try stdout.print("    build      : {}\n", .{gtk_version.comptime_version});
-        try stdout.print("    runtime    : {}\n", .{gtk_version.getRuntimeVersion()});
-        try stdout.print("  - libadwaita : enabled\n", .{});
-        try stdout.print("    build      : {}\n", .{adw_version.comptime_version});
-        try stdout.print("    runtime    : {}\n", .{adw_version.getRuntimeVersion()});
+        if (comptime builtin.os.tag == .linux) {
+            const kernel_info = internal_os.getKernelInfo(alloc);
+            defer if (kernel_info) |k| alloc.free(k);
+            try stdout.print("  - kernel version: {s}\n", .{kernel_info orelse "Kernel information unavailable"});
+        }
+        try stdout.print("  - desktop env   : {s}\n", .{@tagName(internal_os.desktopEnvironment())});
```

Comment:
It might be nice to move the kernel version outside the GTK block in case there's ever a non-GTK linux runtime, but that's minor and can be fixed up later if that ever happens. Or if we extend the kernel version to support macOS/FreeBSD (if it even matters there).

Reply:
Yeah, I assumed that if that ever happens, then it will be moved along with that change, since that would most likely change the API that this is using.